functions/exercise19/scriptSpect.js

```diff
@@ -0,0 +1,30 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals returnElementsLargerThanNumber*/
+
+describe("Returning array elements larger than a number", function () {
+    it("empty array aren't any elements larger than 10", function () {
+        var elementsLargerThanANumber = returnElementsLargerThanNumber(10, []);
+        expect(elementsLargerThanANumber).toEqual([]);
+    });
+    it("in the array whit white space [ ]  aren't any elements larger than 10", function () {
```

Comment:
1. You do not use array with white space here
2. How do you want to compare white space with 10? Is white space smaller or bigger then 10? For me it is obvious that these values can't be compared, so the only thing you may do it is to throw an error - if any item in the array contains non numeric argument, then throw an exception with explanation that argument is incorrect.